### PR TITLE
Upgrade @node-rs/bcrypt to latest version

### PR DESCRIPTION
**Pull request by Brooooooklyn**
No functional changes and breaking changes, add few features:

- Stabilize API
- Support more platforms:
  - Apple darwin arm64
  - Windows x32
  - Android arm64
  - Linux arm7
  - Linux arm64

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
     "parse-server": "bin/parse-server"
   },
   "optionalDependencies": {
-    "@node-rs/bcrypt": "0.4.1"
+    "@node-rs/bcrypt": "^1.1.0"
```
Comment (mtrezza) [inline package.json:119]: Can you please remove the range operator.

@davimacedo Aren't optional dependencies updated automatically by Snyk?

**Review by mtrezza — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Brooooooklyn) [reply package.json]: Resolved

**Review by dplewis — APPROVED** [review]

**Comment by mtrezza** [comment]
@davimacedo Since Snyk runs on your user, aren't optional dependencies updated automatically by Snyk? Or maybe there is a setting to include them that we can turn on?

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0-beta.1](https://github.com/parse-community/parse-server/releases/tag/5.0.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0](https://github.com/parse-community/parse-server/releases/tag/5.0.0)

**Pull request merged**